hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/schema/BaselineAggregation.java

```diff
@@ -0,0 +1,24 @@
+package org.hypertrace.graphql.metric.schema;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+
+public interface BaselineAggregation extends MetricAggregation {
```

Comment:
A couple naming nits:
Could we call this one `MetricBaselineAggregation`

and lowerValue/upperValue lowerBound/upperBound (or similar, I think "value" doesn't convey that it's a range of expectation).